src/test/custom_nodes/node_mock.cpp

```diff
@@ -0,0 +1,29 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "../../custom_node_interface.hpp"
+
+extern "C" int execute(const struct CustomNodeTensor* inputs, int inputsLength, struct CustomNodeTensor** outputs, int* outputsLength, const struct CustomNodeParam* params, int paramsLength) {
+    return 1;
+}
+
+extern "C" int releaseBuffer(struct CustomNodeTensor* output) {
+    return 2;
+}
+
+extern "C" int releaseTensors(struct CustomNodeTensor* outputs) {
+    return 3;
```

Comment:
Not sure if for testing purposes it wouldn't be better to return 0, and only construct special node mock that returns errors.

Reply:
I wanted to differentiate return values to be able to identify these functions in unit test. For integration tests and error handling test purposes we can create separate example library, not a scope of this task.